content/hi/docs/concepts/overview/working-with-objects/owners-dependents.md

```diff
@@ -0,0 +1,22 @@
+---
+title: ओनर और डिपेंडेंट 
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+कुबेरनेट्स में, कुछ {{< glossary_tooltip text="objects" term_id="object" >}} अन्य वस्तुओं के *ओनर* हैं। 
+उदाहरण के लिए, यह {{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}} पॉड्स के एक सेट का ओनर है। ये ओन्ड वाली वस्तुएँ अपने ओनर पे *डिपेंडेंट* हैं।
+
+
+
+
+ओनरशिप [लेबल और सिलेक्टर](/docs/concepts/overview/working-with-objects/labels/) के
+उस मैकेनिज्म से भिन्न है जिसका उपयोग कुछ रिसोर्सेज भी करते हैं। उदाहरण के लिए, एक ऐसी सेवा पर विचार करें
+`एंडपॉइंटस्लाइस` ऑब्जेक्ट बनाता है। सेवा नियंत्रण विमान को अनुमति देने के लिए {{<glosary_tooltip text='labels'term_id='label'>}} का उपयोग करती है
```

Comment:
Typo in the `glossary_tooltip` shortcode name and incorrect formatting is causing the build preview to [fail].(https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65ba7c504707530008932c57)
Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
`EndpointSlice` ऑब्जेक्ट बनाता है। सेवा नियंत्रण विमान को अनुमति देने के लिए {{< glossary_tooltip text="लेबल" term_id='label'>}} का उपयोग करती है
```